src/flop.sol

```diff
@@ -50,16 +50,16 @@ contract Flopper is LibNote {
 
     // --- Data ---
     struct Bid {
-        uint256 bid;
-        uint256 lot;
+        uint256 bid;  // dai paid      [rad]
+        uint256 lot;  // gems for sale [wad]
```

Comment:
I'm not totally satisfied with "gems for sale" here as an accurate description. I'm struggling to come up with a suitably pithy replacement, but maybe consider some of the below options:

"gems bidder will accept"
"gems to receive"
"gems accepted in exchange for bid"
"gems bidder will accept in exchange for bid"
"gems in return"
"gems in return for bid"
"gems in return for bid upon win"
"gems to pay out"

Maybe "gems to receive" works best...

Reply:
bikeshed: I like "gems in return for bid"

Reply:
"gems for sale" was used from commented auction structure in L27-39. 
However, I can update both this line and the original reference in L31. 

I like "gems to receive" as well but don't mind any other description. Just let me know

Reply:
I'm fine with either "gems to receive" or "gems in return for bid". As the PR author, Kenton, the choice is at your discretion when the reviewers are ambivalent :).

Reply:
Going with "gems in return for bid"